index.coffee

```diff
@@ -69,18 +12,25 @@ module.exports =
       default: ''
 
   activate: ->
-    prefix = 'linter-rubocop.'
     @subscriptions = new CompositeDisposable
-    @subscriptions.add atom.config.observe "#{prefix}executablePath",
-      (args) => @executablePath = if args then args.split ' ' else ['rubocop']
-    @subscriptions.add atom.config.observe "#{prefix}additionalArguments",
-      (args) => @additionalArguments = if args then args.split ' ' else []
+    @subscriptions.add atom.config.observe('linter-rubocop.executablePath', (executablePath) =>
+      @executablePath = executablePath
```

Comment:
@caseywebdev could you test rbenv now? Because having `/usr/local/bin/chruby-exec` as an executable path and `ruby -- rubocop` as arguments doesn't work for me. I get
```
Error: chruby-exec: COMMAND required
Error: chruby-exec: COMMAND required
  at exit (/Users/mrfoto/.atom/packages/linter-rubocop/node_modules/atom-linter/lib/helpers.coffee:32:24)
  at triggerExitCallback (/Applications/Atom.app/Contents/Resources/app.asar/src/buffered-process.js:213:47)
  at /Applications/Atom.app/Contents/Resources/app.asar/src/buffered-process.js:227:18
  at Socket.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/src/buffered-process.js:98:18)
  at emitOne (events.js:82:20)
  at Socket.emit (events.js:166:7)
  at Pipe.close (net.js:464:12)
```